benchmarks/src/benchmark.cpp

```diff
@@ -2124,7 +2124,7 @@ void Benchmark::run_convert_utf16_to_utf8_utfcpp(size_t iterations) {
 void Benchmark::run_convert_utf8_to_utf32_utfcpp(size_t iterations) {
     const char*  data = reinterpret_cast<const char*>(input_data.data());
     const size_t size = input_data.size();
-    std::string data_string = std::string(data, size);
+    std::string_view data_string = std::string_view(data);
```

Comment:
Same question here... why not provide the std::string_view constructor with the size parameter? (I guess that the question applies throughout.)